codegen/library.xml

```diff
@@ -15,48 +15,40 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 -->
-    <!-- Configure the database connection here -->
-    <jdbc>
-        <driver>com.mysql.jdbc.Driver</driver>
-        <url>jdbc:mysql://localhost:3306/killbill</url>
-        <user>killbill</user>
-        <password>killbill</password>
-    </jdbc>
-
-    <generator>
-        <!-- The default code generator. You can override this one, to generate your own code style
+  <!-- Configure the database connection here -->
+  <jdbc>
+    <driver>com.mysql.jdbc.Driver</driver>
+    <url>jdbc:mysql://192.168.99.100:33066/killbill</url>
+    <user>root</user>
+    <password>honeybadger</password>
```

Comment:
can you put this back to what it was, with localhost:3306 and the killbill user/root

Reply:
Addressed in the latest force pushed commit.